rules/common.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    mocha: true,
+    jasmine: true,
+    node: true,
+    commonjs: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+  rules: {
+    // -- Error
+    'no-shadow': 2,
+    'default-case': 2,
+
+    // -- Warning
+    'class-methods-use-this': 1,
+    'no-else-return': 1,
+    'no-param-reassign': 1,
+    'no-underscore-dangle': 1,
+    'no-unused-vars': 1,
+    semi: [1, 'always'],
+    'import/no-unresolved': 1,
+    'import/prefer-default-export': 1,
+    'prefer-destructuring': 1,
+    'lines-between-class-members': 1,
+    'consistent-return': 1,
+    'max-lines': ['warn', { max: 500, skipBlankLines: true, skipComments: true }],
+    'max-classes-per-file': ['warn', 3],
+    'max-lines-per-function': ['warn', { max: 75 }],
```

Comment:
Since you can pass in an array, can you specify something like: `['error', { max: 75 }, 'warn', { max: 25 }]`? I can't tell if it's just an array...because (and should be a hash with a severity key), or supports multiple values.

Reply:
This is actually a really good idea! I looked into it [here](https://github.com/eslint/eslint/issues/3427) and it isn't a thing in vanilla ESLint, but we could perhaps write an ESLint plugin with warning-error thresholds at some point.